components/tooltip/tooltip.js

```diff
@@ -14,12 +15,19 @@ const pointerOverhang = 7; /* how far the pointer extends outside the content */
 const pointerRotatedLength = Math.SQRT2 * pointerLength;
 const pointerRotatedOverhang = ((pointerRotatedLength - pointerLength) / 2) + pointerOverhang;
 
-const pointerGap = 6; /* spacing between pointer and target */
+const pointerGap = 0; /* spacing between pointer and target */
 const defaultViewportMargin = 18;
 const contentBorderRadius = 6;
 const contentBorderSize = 1;
 const contentHorizontalPadding = 15;
 
+const interactiveElements = {
+	'input': true,
+	'select': true,
+	'textarea': true,
+	'button': true
```

Comment:
What about `img`? I know it's not an interactive element, but I was assuming that because screen readers recognize it and is has a label (`alt`) that `aria-describedby` would work.

Reply:
There likely are a few more elements that could be added to the list. I'm happy to test a bunch of different elements if we're interested in expanding it a bit.

I left it with these ones because they're ones I'm confident are consistent in all cases regardless of attributes and I also foresee them getting used in the future when tooltip gets integrated into other `core` components like `d2l-button` and `d2l-input`.

Reply:
I mostly bring up `img` and `a` because of the work Margaret did in the monolith to add tooltip support to the legacy `<Image>` control -- which can associate the tooltip with both of those depending on the setup.

Reply:
Unfortunately it looks like `img` won't work. `aria-describedby` doesn't trigger announcments for `img` tags even if they have `alt` text and a `tabindex`

Reply:
Oh dear, that might ruin Margaret's use case then. Does `aria-labelledby` work? Although that'll just totally replace the `alt` likely.

Reply:
With `img` I think the `announce` helper is the best we'll be able to do. `aria-labelledby` and `aria-describedby` are super inconsistent with static content. [I found this article which seems to be accurate for the most part](http://www.davidmacd.com/blog/does-aria-label-override-static-text.html)

Reply:
Seems that way yeah. Cool well thanks for investigating and all the testing!

Reply:
With the announce helper it reads:

> `${alt text}`
graphic
`${tooltip message}`

Reply:
Perfect. 🥇